waiter/integration/waiter/websocket_integration_test.clj

```diff
@@ -153,6 +160,62 @@
         (finally
           (delete-service waiter-url waiter-headers))))))
 
+(deftest ^:parallel ^:integration-slow test-fail-to-stream-large-responses
+  (testing-using-waiter-url
+    (let [^WebSocketClient websocket-client (websocket-client-factory)
+          waiter-settings (waiter-settings waiter-url)
+          {:keys [ws-max-binary-message-size ws-max-text-message-size]} (:websocket-config waiter-settings)
+          ws-max-binary-message-size' (+ 2048 ws-max-binary-message-size)
```

Comment:
What's the point of adding 2048 here, resetting the max, and then adding another 1024 when we send the message?  Why not use the configured max and just add the 1024 for the message without `update-max-message-sizes`?